File: ApiSuvesaPos/Negocio/Modelo/cxc/devoluciones_ventas/articulos_ventas_devueltos.vb
Namespace Modelo
    Public Class articulos_ventas_devueltos
        Public Property Consecutivo As Long
        Public Property Devolucion As Long
        Public Property Codigo As Long
        Public Property Descripcion As String
        Public Property Cantidad As Decimal
        Public Property CantVet As Decimal
        Public Property CantBod As Decimal
        Public Property Precio_Costo As Decimal
        Public Property Precio_Base As Decimal
        Public Property Precio_Flete As Decimal
        Public Property Precio_Otros As Decimal
        Public Property Precio_Unit As Decimal
        Public Property Descuento As Decimal
        Public Property Monto_Descuento As Decimal
        Public Property Impuesto As Decimal
        Public Property Monto_Impuesto As Decimal
        Public Property SubtotalGravado As Decimal
        Public Property SubTotalExcento As Decimal
        Public Property SubTotal As Decimal
        Public Property Id_Art_Venta As Integer
        Public Property id_articulo_V As String
        Public Property devuelto As Boolean
        Public Property empaquetado As Boolean
        Public Property recibido As Boolean
    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/cxc/devoluciones_ventas/articulos_ventas_devueltos.cs
namespace Modelo
{
    public class articulos_ventas_devueltos
    {
        public long Consecutivo { get; set; }
        public long Devolucion { get; set; }
        public long Codigo { get; set; }
        public string Descripcion { get; set; }
        public decimal Cantidad { get; set; }
        public decimal CantVet { get; set; }
        public decimal CantBod { get; set; }
        public decimal Precio_Costo { get; set; }
        public decimal Precio_Base { get; set; }
        public decimal Precio_Flete { get; set; }
        public decimal Precio_Otros { get; set; }
        public decimal Precio_Unit { get; set; }
        public decimal Descuento { get; set; }
        public decimal Monto_Descuento { get; set; }
        public decimal Impuesto { get; set; }
        public decimal Monto_Impuesto { get; set; }
        public decimal SubtotalGravado { get; set; }
        public decimal SubTotalExcento { get; set; }
        public decimal SubTotal { get; set; }
        public int Id_Art_Venta { get; set; }
        public string id_articulo_V { get; set; }
        public bool devuelto { get; set; }
        public bool empaquetado { get; set; }
        public bool recibido { get; set; }
    }
}
